api/uniter/unit_test.go

```diff
@@ -684,6 +684,32 @@ func (s *unitSuite) TestWatchActionNotificationsMoreResults(c *gc.C) {
 	c.Assert(err.Error(), gc.Equals, "expected 1 result, got 2")
 }
 
+func (s *unitSuite) TestWatchUpgradeSeriesNotifications(c *gc.C) {
+	watcher, err := s.apiUnit.WatchUpgradeSeriesNotifications()
+	c.Assert(err, jc.ErrorIsNil)
+
+	notifyWatcher := watchertest.NewNotifyWatcherC(c, watcher, s.BackingState.StartSync)
+	defer notifyWatcher.AssertStops()
+
+	// Why do I have two initial events?!?
```

Comment:
There are apparently two initial events coming out of this puppy. Why?

Reply:
Did you find out why this is happening?  I noticed the TestWatchActionNotificationsMoreResults() is testing that only 1 watch notification happens.

Reply:
Yes, that is what I mean above. No I don't know why, its innocuous (since the Uniter will just loop twice), but I don't know why its sending two initial events.

Reply:
All watchers send an initial event to trigger the checking of initial state. Many times this isn't useful, and you'll see comments like "consume initial event".

Reply:
OK, fixed now.